docs/docs/91-migrations.md

```diff
@@ -21,6 +21,7 @@ Some versions need some changes to the server configuration or the pipeline conf
 - Removed `WOODPECKER_DOCS` config variable
 - Renamed `link` to `url` (including all API fields)
 - Deprecated `CI_COMMIT_URL` env var, use `CI_PIPELINE_FORGE_URL`
+- Migrated from Bitbucket Stash support to Bitbucket Datacenter / server support, rename env `WOODPECKER_STASH` to `WOODPECKER_BITBUCKET_DC` and webhooks need to be re-installed (e.g., using "repair" on the project in Woodpecker)
```

Comment:
True we are getting pretty far from the original implementation - I'll remove the migration part. I kept it because there were some issue way back that concerned the Bitbucket support gone missing in this major version - so if that guy were to actually jump to the new implementation he would have to "migrate".